csrc/net/ncnn/ncnn_net.cpp

```diff
@@ -21,7 +21,20 @@ Result<void> NCNNNet::Init(const Value& args) {
   auto& context = args["context"];
   device_ = context["device"].get<Device>();
   stream_ = context["stream"].get<Stream>();
-
+#ifdef __aarch64__
+#ifndef FP16_ACCELERATE
+  net_.opt.use_fp16_storage = false;
+  net_.opt.use_fp16_arithmetic = false;
+#endif
+#endif
+#ifdef NCNN_NET_VULKAN
```

Comment:
Did that means fp16 will be disabled automatically when valkan is enabled?

Reply:
No. I will consider reorganize the fp16 and vulkan options.